templates/paragraph--airtable--training-upcoming.html.twig

```diff
@@ -0,0 +1,127 @@
+ *   and methods for the paragraph object.
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{%
+  set classes = [
+    'paragraph',
+    'paragraph--type--' ~ paragraph.bundle|clean_class,
+    view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
+    not paragraph.isPublished() ? 'paragraph--unpublished',
+    'rh-training-upcoming'
+  ]
+%}
+{%
+  set airtableConfig = {
+    "table": "tbl76xG6T8QA20lqn",
+    "view": "viwEAwOfMreLa7eVt",
+    "filters": {
+      "Category": {
+        "name": "Categories"
+      },
+      "Series": {
+        "name": "Series"
+      }
+    },
+    "format": {
+      "Event Start Date": {
+        "type": "Date",
+        "options": {
+          "date": {
+            weekday: "long",
+            year: "numeric",
+            month: "long",
+            day: "numeric"
+          },
+          "time": {
+            hour: "2-digit",
+            minute: "2-digit"
+          }
+        }
+      },
+      "Event End Date": {
+        "type": "Date",
+        "options": {
+          "time": {
+            hour: "2-digit",
+            minute: "2-digit"
+          }
+        }
+      }
+    }
+  }
+%}
+{% block paragraph %}
+  <div{{ attributes.addClass(classes) }}>
+    {% block content %}
+      {# Filters #}
+      {#<div id="airtable-list-filters"></div>#}
+      <div id="airtable-list" data-config='{{ airtableConfig | json_encode }}'>
+        <p id="airtable-list-loader">Loading upcoming training...</p>
+        <p id="airtable-list-no-results" style="display: none;">There are no results matching your current criteria.</p>
+        <ul id="airtable-list-record-wrapper">
+        </ul>
+      </div>
+
+      <template id="airtable-list-records-template">
```

Comment:
I think this is the first time I've seen the template tag in real use. Bravo.

Reply:
Yeah I hadn't heard of it before, but since I'm limited in what I can do it works. :)